projects/snapshot_bot/snapshot_bot.cc

```diff
@@ -513,13 +416,13 @@ class RobotFSM : FSM<State>::StateHandler
                 // If A is pressed
                 if(m_Joystick.isPressed(HID::JButton::A)) {
                     // Subtract time we've already moved for from drive time
-                    m_DriveTime -= m_MoveStopwatch.elapsed();
+                    m_DriveTime -= static_cast<millisecond_t>(m_MoveStopwatch.elapsed());
```

Comment:
how come this needs a cast? Doesn't the stopwatch also return a time unit?

Reply:
It doesn't, though maybe it should!